shared/src/msi-installer/shared/EnvironmentVariables.wxs

```diff
@@ -21,14 +21,6 @@
     </ComponentGroup>
   
     <ComponentGroup Id="Shared.EnvironmentVariables.IIS" Directory="INSTALLFOLDER">
-      <Component Id="Shared.Registry.EnvironmentVariables.W3SVC" Guid="{702DB265-F33E-47F4-A6B0-E21FA0FC21C1}" Win64="$(var.Win64)">
-        <CreateFolder/>
```

Comment:
What does createFolder does here? It creates the group in the registry for `Shared.Registry.EnvironmentVariables.W3SVC `if it didn't exist? Just to be sure I understand what I'm approving? (I looked at the documentation but I'm not sure so I'd rather ask)